pkg/lockfile/parse-nuget-lock.go

```diff
@@ -18,11 +20,15 @@ type NuGetLockfile struct {
 }
 
 const NuGetEcosystem Ecosystem = "NuGet"
+const projectDependencyType = "Project"
```

Comment:
```suggestion
const ProjectDependencyType = "Project"
```

Reply:
In go when you have a capital letter as first character you export the variable
I don't want to do that here as it have no meaning outside the nuget lockfile parser

Reply:
:todayIlearnt: thanks for the clarification!